### ISSUE TITLE
felogin not working?

### ISSUE THREAD

**Issue by dabis-eu** [benjaminkott/bootstrap_package#1189]
Seems like the Package only  contains the old p1 style frontendlogin.html but is missing the extbase version... or am i overlooking something?
thanks in advance

**Comment by dahaupt** [comment]
@dabis-eu I can confirm your assumption. I had to override the template as well in a recent project.

**Comment by dabis-eu** [comment]
Yeah customized it aswell, not a big deal but thought i might bring it up
